test/e2e/far_e2e_test.go

```diff
@@ -309,14 +309,28 @@ func wasFarTaintAdded(nodeName string) {
 	log.Info("FAR taint was added", "node name", node.Name, "taint key", farTaint.Key, "taint effect", farTaint.Effect)
 }
 
-// checkFarLogs gets the FAR pod and checks whether its logs have logString
-func checkFarLogs(logString string) {
+// buildExpectedLogOutput returns a string with a node identifier and a success message for the reboot action
+func buildExpectedLogOutput(nodeName, sucesssMessage string) string {
+	expectedString := "\"Node name\": \"" + nodeName + "\", \"Response\": \"" + sucesssMessage
```

Comment:
is there a reason not to use `fmt.Sprintf`? I should be easier to read

Reply:
Good point